### ISSUE TITLE
Image chooser modal form fields aren’t translatable

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#6737]
### Issue Summary

In the image chooser modal, some of the fields are missing translations due to them not being marked for translations:

- The Collection field on the Upload form. I wouldn’t be surprised if this was the case for all Upload forms, as well as the one for documents, as they all inherit from the same base form.
- The Format and Alt text fields on the `ImageInsertionForm`.

![translations-chooser-upload](https://user-images.githubusercontent.com/877585/106126793-c038e280-6155-11eb-86cd-2d9247890592.png)

![translations-chooser-format](https://user-images.githubusercontent.com/877585/106126803-c3cc6980-6155-11eb-90c7-778e9fe2cac8.png)

### Steps to Reproduce

1. Use Wagtail in any other language than english
2. Open the image chooser modal from a rich text editor
3. Note the field labels are in english.

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

All of Wagtail’s admin UI defined in core should be translate-able.

* I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: no

### Technical details

* Wagtail version: Wagtail v.2.13a0

**Comment by thibaudcolas** [comment]
Fixed in #6730 for the "select format" step fields. The Collection issue is more widespread so I’ll open a separate PR just for that.
